src/H5FDsubfiling/H5FDioc_err.h

```diff
@@ -0,0 +1,290 @@
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+ * Copyright by The HDF Group.                                               *
+ * All rights reserved.                                                      *
+ *                                                                           *
+ * This file is part of HDF5.  The full HDF5 copyright notice, including     *
+ * terms governing use, modification, and redistribution, is contained in    *
+ * the COPYING file, which can be found at the root of the source code       *
+ * distribution tree, or in https://support.hdfgroup.org/ftp/HDF5/releases.  *
+ * If you do not have access to either file, you may request a copy from     *
+ * help@hdfgroup.org.                                                        *
+ * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+
+/*
+ * Error handling for the HDF5 IOC VFD
+ */
+
+#ifndef H5FDioc_err_h
+#define H5FDioc_err_h
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <errno.h>
+
+#include "H5Epublic.h"
+
+extern hid_t H5FDioc_err_stack_g;
+extern hid_t H5FDioc_err_class_g;
+
+#define H5FD_IOC_ERR_CLS_NAME "HDF5 IOC VFD"
+#define H5FD_IOC_ERR_LIB_NAME "HDF5 IOC VFD"
+#define H5FD_IOC_ERR_VER      "1.0.0"
+
+#define SUCCEED 0
+#define FAIL    (-1)
```

Comment:
We really need to name these H5_SUCCEED and H5_FAIL and expose them in H5public.h